transcription-handler.go

```diff
@@ -66,7 +66,7 @@ func (h *handlers) SubmitJob(c *gin.Context) {
 	}
 
 	if job.Model == "" {
-		job.Model = "large-v3"
+		job.Model = "large-v2"
```

Comment:
Why was this downgraded?

Reply:
I was testing something, but it appears that it is not used at all. Thanks for spotting it'll fix it up.

Reply:
I actually respect the value passed in now but filter it on known good values. Also updated all defaults to V3